OpenRA.Mods.Common/Widgets/ObserverProductionIconsWidget.cs

```diff
@@ -75,15 +75,16 @@ public override void Draw()
 				var rsi = actor.Traits.Get<RenderSpritesInfo>();
 				var icon = new Animation(world, rsi.GetImage(actor, world.Map.SequenceProvider, race));
 				icon.Play(actor.Traits.Get<TooltipInfo>().Icon);
+				var bi = actor.Traits.GetOrDefault<BuildableInfo>();
```

Comment:
Use `Get`.  We are guaranteed that it won't fail because  non-`Buildable` actors can't show up here.